src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/publication/FileGenerationTest.java

```diff
@@ -178,7 +188,7 @@ void shouldGenerateAnExcelOnPublicationUpload() throws Exception {
         assertNotNull(publicationFileSizes.getExcel(), "Excel has not been generated");
 
```

Comment:
Some of these tests of SYSTEM header as TRUE and Requester ID is also set. I think only one of these is needed in each test